### ISSUE TITLE
Fix accessibility issues 

### ISSUE THREAD

**Issue by mgifford** [ResponsibleTechWork/RespTechWork-website#3]
![Screen Shot of WAVE evaluation of home page](https://user-images.githubusercontent.com/116832/130334146-48c45d55-da6d-4548-b4cd-ab45bcbc9f9e.png)

There are likely other issues on subsequent pages, but would be important to demonstrate that the site at least avoids accessibility errors via automated testing.

**Comment by ialja** [comment]
Yes, this is one of the things we'd like to address as we release v1 of the framework. For now, the website still largely uses theme defaults, and we're aware of the minor contrast issues, mostly on buttons. We're also planning on increasing the base font size and improving the readability of pages with longer text. Any other suggestions are most welcome!

**Comment by ialja** [comment]
Quick update: we addressed all contrast errors and improved general readability with #6. I'm leaving this issue open because we still plan on addressing alerts that pop up with automated testing and improve mobile accessibility.
